<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestForm</class>
 <widget class="QDialog" name="TestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #7B7C81;
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="btnAnswer1">
     <property name="font">
      <font>
       <family>Sitka Heading</family>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
   background-color: #7B586B; 
   color: white; 
   border: 2px solid #1D0F0F; 
   border-radius: 8px; 
   padding: 3; }

QPushButton:pressed { 
	background-color: #D4DBE2;
 }</string>
     </property>
     <property name="text">
      <string>Answer1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnAnswer2">
     <property name="font">
      <font>
       <family>Sitka Heading</family>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
   background-color: #7B586B; 
   color: white; 
   border: 2px solid #1D0F0F; 
   border-radius: 8px; 
   padding: 3; }

QPushButton:pressed { 
	background-color: #D4DBE2;
 }</string>
     </property>
     <property name="text">
      <string>Answer2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnAnswer3">
     <property name="font">
      <font>
       <family>Sitka Heading</family>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
   background-color: #7B586B; 
   color: white; 
   border: 2px solid #1D0F0F; 
   border-radius: 8px; 
   padding: 3; }

QPushButton:pressed { 
	background-color: #D4DBE2;
 }</string>
     </property>
     <property name="text">
      <string>Answer3</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="lWord">
     <property name="font">
      <font>
       <family>Sitka Heading</family>
       <pointsize>40</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #333333; /* Цвет текста */
    border: 1px solid #CCCCCC; /* Граница */
    border-radius: 5px; /* Скругление углов */
    padding: 5px; /* Отступы вокруг текста */
    text-align: center; /* Выравнивание текста по центру */
}</string>
     </property>
     <property name="text">
      <string>Example</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
